django/control/api/views.py

```diff
@@ -90,3 +92,15 @@ def post(self, request):
                 )
 
         return Response(status=status.HTTP_200_OK)
+
+
+class ScriptsEndpoint(views.APIView):
+    def get(self, request):
+        res = []
+        for module in getmembers(scripts, ismodule):
+            print(module)
+            for script in getmembers(module[1], isfunction):
```

Comment:
What's module[1] and module[2]?
Can you write `for <blablablabla>, <pookie> in getmembers(scripts, ismodule)` above?